src/search/pwm.jl

```diff
@@ -315,3 +354,42 @@ function show_nuc_matrix(io::IO, m::Union{PFM{S},PWM{S}}) where S<:Union{DNA,RNA
         end
     end
 end
+
+struct PWMSearchQuery{S,T,R<:Real}
+    pwm::PWM{S,T}
+    threshold::R
+end
+
+function PWMSearchQuery(pwm::PWM{S,T}, threshold::R) where {S,T,R<:Real}
+    return PWMSearchQuery{S,T,R}(pwm, threshold)
+end
```

Comment:
This function is redundant - it is automatically generated by the struct definition. Can be deleted